code/weapon/beam.cpp

```diff
@@ -3044,32 +3047,25 @@ void beam_handle_collisions(beam *b)
 		// add to the recent collision list
 		r_coll[r_coll_count].c_objnum = target;
 		r_coll[r_coll_count].c_sig = Objects[target].signature;
-		r_coll[r_coll_count].c_stamp = -1;
 		r_coll[r_coll_count].cinfo = b->f_collisions[idx].cinfo;
 		r_coll[r_coll_count].quadrant = -1;
 		r_coll[r_coll_count].is_exit_collision = 0;
 		
+
+		// get the real damage that should be done
+		damage_time_mod = (flFrametime * 1000.0) / i2fl(BEAM_DAMAGE_TIME);
```

Comment:
You either need to change `1000.0` to `1000.0f`, or wrap the whole right side of this equation in a conversation to `float` to avoid the warning stopping AppVeyor from passing this PR.